### PR TITLE
Pn without preview

### PR DESCRIPTION
**Pull request by RyanRory**
This is a version of PN without preview. The content of PN just includes the message count.

### CODE REVIEW

**Comment by gmbnt** [comment]
Documentation on the difference between the two methods in `LokiPushNotificationManager` would be nice. Also, there's a bit of overlap between them that you could factor out.

Looks good otherwise :)

**Comment by gmbnt** [comment]
Also, `LokiPushNotificationManager` doesn't hold any state so you could do without the singleton and just make everything `static`.

**Comment by RyanRory** [comment]
I'll refactor the `LokiPushNotificationManager` in the PN with preview PR.

File: SignalServiceKit/src/Messages/TypingIndicatorMessage.swift [hunk 1]
```diff
@@ -99,6 +99,11 @@ public class TypingIndicatorMessage: TSOutgoingMessage {
     public override func shouldBeSaved() -> Bool {
         return false
     }
+    
+    @objc
+    public override var ttl: UInt32 {
+        return UInt32(5 * kMinuteInMs)
```
Comment (gmbnt) [inline SignalServiceKit/src/Messages/TypingIndicatorMessage.swift]: Let's make this one minute to match other message types.

**Pull request merged**